datasets/acronym_identification/README.md

```diff
@@ -0,0 +1,181 @@
+---
+annotations_creators:
+- expert-generated
+language_creators:
+- found
+languages:
+- en
+licenses:
+- mit
+multilinguality:
+- monolingual
+size_categories:
+- 10K<n<100K
+source_datasets:
+- original
+task_categories:
+- structure-prediction
+task_ids:
+- structure-prediction-other-acronym-identification
+---
+
+# Dataset Card Creation Guide
+
+## Table of Contents
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-instances)
+  - [Data Splits](#data-instances)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:** https://sites.google.com/view/sdu-aaai21/shared-task
+- **Repository:** https://github.com/amirveyseh/AAAI-21-SDU-shared-task-1-AI
+- **Paper:** https://arxiv.org/pdf/2010.14678v1.pdf
+- **Leaderboard:** https://competitions.codalab.org/competitions/26609
+- **Point of Contact:** [More Information Needed]
+
+### Dataset Summary
+
+[More Information Needed]
+
+### Supported Tasks and Leaderboards
+
+[More Information Needed]
+
+### Languages
+
+[More Information Needed]
+
+## Dataset Structure
+
+### Data Instances
+
+```
+DatasetDict({
```

Comment:
In this section we expect to see an example of data sample, as the one you added in the Data Field section. Can you move it in this section instead please ?

In the Data Field section you can add a brief description of each one of the data fields id, tokens labels, and also explain the BIO format used for labels